.cicd/platforms/amazon_linux-2-unpinned.dockerfile

```diff
@@ -5,7 +5,7 @@ RUN yum update -y && \
     yum install -y which git sudo procps-ng util-linux autoconf automake \
     libtool make bzip2 bzip2-devel openssl-devel gmp-devel libstdc++ libcurl-devel \
     libusbx-devel python3 python3-devel python-devel libedit-devel doxygen \
-    graphviz clang patch
+    graphviz clang patch vim-common jq
```

Comment:
You shouldn't need `jq` in the `release/1.8.x` version of this test doesn't use `jq`. But it is okay to keep it if you want for other reasons.